flocker/node/agents/test/test_blockdevice.py

```diff
@@ -169,12 +169,10 @@
 
 DESIRED_DATASET_ATTRIBUTE_STRATEGIES = {
     'dataset_id': uuids(),
-    'maximum_size': integers(min_value=0).map(
-        lambda n: (
-            LOOPBACK_MINIMUM_ALLOCATABLE_SIZE +
-            n * LOOPBACK_ALLOCATION_UNIT
-        )
-    ),
+    'maximum_size': integers(
```

Comment:
Yeah, that does make sense.
I'm also inclined to make this a fixed value `just(LOOPBACK_MINIMUM_ALLOCATABLE_SIZE)` 
....otherwise it seems like this is an in appropriate place to test for the ability of the loopback driver to handle a range of device sizes.
It might make more sense if we supported resizing or if the tests weren't actually calling the backend.